### PR TITLE
CR for 20513 - Adapt DDE control of eyes to be predominantly open or closed

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1d7420d0dc908cabca046ceba5b4b8c6836710bf available at: https://gist.github.com/4d4d6ea84543f6faeb81<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4718.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4718.dmg

File: interface/src/devices/DdeFaceTracker.cpp [hunk 1]
```diff
@@ -403,6 +388,88 @@ void DdeFaceTracker::decodePacket(const QByteArray& buffer) {
...
+                float velocity = fabs(eyeBlinks[i] - _lastEyeBlinks[i]) / _averageMessageTime;
+                float velocityFilter = glm::clamp(velocity * BLINK_VELOCITY_FILTER_STRENGTH, 0.0f, 1.0f);
+                _filteredEyeBlinks[i] = velocityFilter * eyeBlinks[i] + (1.0f - velocityFilter) * _filteredEyeBlinks[i];
+                _lastEyeBlinks[i] = eyeBlinks[i];
+            }
+        }
+
+        // Finesse EyeBlink values
+        float eyeCoefficients[2];
+        for (int i = 0; i < 2; i += 1) {
```
Comment (ZappoMan) [inline interface/src/devices/DdeFaceTracker.cpp]: this is a very strange looking construct... why not i++

**Comment by hifi-gustavo** [comment]
Build console output for commit: 81e23eab28ae151f6ddd5f16bd222787663f7e29 available at: https://gist.github.com/4d4d6ea84543f6faeb81<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4718.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4718.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e45ecbe2c8d5b101e23b1ab1c504aee593e8ef97 available at: https://gist.github.com/4d4d6ea84543f6faeb81<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4718.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4718.dmg

**Comment by PhilipRosedale** [comment]
nice improvement!

**Pull request merged**